util/argo/managedfields/managed_fields.go

```diff
@@ -28,12 +28,14 @@ func Normalize(live, config *unstructured.Unstructured, trustedManagers []string
 
 	liveCopy := live.DeepCopy()
 	configCopy := config.DeepCopy()
+	normalized := false
+
 	results, err := newTypedResults(liveCopy, configCopy, pt)
+	// error might happen if the resources are not parsable and so cannot be normalized
 	if err != nil {
-		return nil, nil, fmt.Errorf("error building typed results: %w", err)
+		return liveCopy, configCopy, nil
```

Comment:
As we are not returning the error anymore, it might be helpful to log the error message here for debugging.

Reply:
Agreed -  I've added logging! Used debug level since this method will be executed on each app reconciliation and might produce a lot of noise in logs.

Reply:
Debug sounds good!!